Rakefile

```diff
@@ -100,6 +135,16 @@ def storage
   )
 end
 
+def with_sentry_dsn
```

Comment:
The sentry_dsn is a secret so this piece of code makes sure the value is defined in the `SentryDsn.swift` file when packaging a new release of Tuist.